package.json

```diff
@@ -38,13 +38,15 @@
     "@types/mocha": "^10.0.1",
     "@typescript-eslint/eslint-plugin": "^7.0.1",
     "@typescript-eslint/parser": "^7.0.1",
+    "asana": "^1.0.2",
```

Comment:
I think this Asana version might be outdated - we're using a version straight from github in other projects (e.g. https://github.com/duckduckgo/duckduckgo-privacy-extension/blob/main/package.json#L42).

Reply:
Looks like this version is a year old, and there is now a v3 version out: https://www.npmjs.com/package/asana?activeTab=versions

Not sure if the API is the same between major version though - when working with this library in the past I've seen core API changes happen between versions.

Reply:
I first started with the latest version, but there were some incompatibilities. I didn't look into them deeply, figured it was ok to use this version since the other asana workflow is [using the same](https://github.com/sammacbeth/action-asana-sync/blob/v4/package-lock.json#L2024). Would be nice to keep it up to date, but perhaps it could be a follow up and cover both automations?